lib/bibliothecary/parsers/rubygems.rb

```diff
@@ -50,7 +49,16 @@ def self.parse_gemspec(file_contents)
         parse_ruby_manifest(manifest)
       end
 
-
+      def self.set_related_paths_field(by_dirname_dest, by_dirname_source)
```

Comment:
It's probably worth some comments about how this varies from the base case -- I'm also a little wary that we'll make a change to one and not the other as it's not clear that there's overriding happening here

Reply:
Yeah, you're right - I'll rearrange this to be more declarative